telluric/georaster.py

```diff
@@ -736,6 +758,19 @@ def height(self):
         """Raster height."""
         return int(self.shape[1])
 
+    @property
+    def blockshapes(self):
+        """Raster all bands block shape."""
+        if self._blockshapes is None:
```

Comment:
when we have this kind of caching we need to ask ourselves what are implication in the Mutable implementation, 

Is this behavior still valid there?